spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe ApplicationHelper do
+  include ApplicationHelper
+  include ViewHelper
+  include GovukComponentsHelper
+  include GovukLinkHelper
+
+  describe "#enrichment_error_link" do
+    context "with a course" do
+      before do
+        @provider = Provider.new(build(:provider).attributes)
+        @course = Course.new(build(:course).attributes)
+      end
+
+      it "returns correct content" do
+        expect(enrichment_error_link(:course, "about_course", "Something about the course"))
+          .to eq("<div class=\"govuk-inset-text app-inset-text--narrow-border app-inset-text--error\"><a class=\"govuk-link\" href=\"/publish/organisations/#{@provider.provider_code}/#{@course.recruitment_cycle_year}/courses/#{@course.course_code}/about?display_errors=true#publish-course-information-form-about-course-field-error\">Something about the course</a></div>")
+      end
+    end
+  end
+
+  describe "#enrichment_summary" do
+    let(:summary_list) { GovukComponent::SummaryListComponent.new }
+
+    subject { render(summary_list) }
```

Comment:
I think the subject is no longer `ApplicationHelper`

Reply:
https://github.com/DFE-Digital/teacher-training-api/pull/2587#discussion_r837638161